src/js/features/conversations.js

```diff
@@ -34,6 +33,9 @@ function Conversations(timeout) {
     if (window.App && App.__container__.lookup('service:whispers-shim')) {
         var whisperShim = App.__container__.lookup('service:whispers-shim');
         whisperShim.on('whisper', _self.onWhisper);
+        whisperShim.on('thread', function(data) {
+            bttv.ws.joinConversation(data.id);
```

Comment:
You can join on thread, but you should broadcast on whisper.

Additionally you should handle the case of parting convos too.